### ISSUE TITLE
formatCQL modifies params

### ISSUE THREAD

**Issue by simpleseeker** [simplereach/helenus#103]
I have a variable that is an array that I would like to reuse for different queries. After a little digging, I undercover formatCQL modifying my param variable, which prevents me from reusing it.

**Comment by devdazed** [comment]
yeah, the code goes through and pops off each of the params.  I'll make a note of this in the latest CQL revamp.

**Comment by asilvas** [comment]
This hit us as well. Caught us off guard as we use consistency fallback levels for some failures to be re-executed. So our hack is to copy arrays for high-consistency requests, which is wasteful.

**Comment by asilvas** [comment]
Was this fixed or not going to fix?

**Comment by devdazed** [comment]
ah, I guess this is a bug, I'll reopen.
